game.js

```diff
@@ -1,12 +1,179 @@
-var a = 0
-document.getElementById('button').onclick = Click
-function Click() {
-  document.getElementById('button').innerHTML = a
-  a = a + 1
-  for (var i = 0; i < a; i++){
-    var li = document.createElement("LI");
-    var textnode = document.createTextNode("Cica");
-    li.appendChild(textnode)
-    document.getElementById("myList").appendChild(li);
+// GLOBALS
+// GLOBALS of GRID.
+maxColumns = 10;
```

Comment:
javascript nagyon megbocsato, de ez egy megbizhato szabaly, ami alol nem kell, h legyen kivetel: mindig jeloljuk, h valtozot definialunk. (egyelore var-ral kesobb majd masokkal is :))